### PR TITLE
Connect map to home page

### PR DESCRIPTION
**Pull request by romawht22**
Connected leaflet map to home page and created a function that prints map coordinates on mouse click.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -11,9 +11,11 @@
     "@testing-library/react": "^12.0.0",
     "@testing-library/user-event": "^13.2.1",
     "husky": "^7.0.4",
+    "leaflet": "^1.7.1",
     "lint-staged": "^12.3.7",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
+    "react-leaflet": "^3.2.5",
```
Comment (steadyGuy) [inline frontend/package.json:18]: Just a question. Do we really need an additional library-wrapper like _react-leaflet_ (maybe we can do with only _leaflet_ package?)

File: frontend/src/components/Map/index.tsx [hunk 2]
```diff
@@ -1,8 +1,25 @@
 import React from 'react';
-import { StyledMap } from './styles';
+import { MapContainer, TileLayer, useMapEvents } from 'react-leaflet';
 
 function Map() {
-  return <StyledMap sx={{ mt: 20 }}>Map</StyledMap>;
+  function CoordsFinder() {
+    useMapEvents({
+      click(e) {
+        console.log(e.latlng);
```
Comment (steadyGuy) [inline frontend/src/components/Map/index.tsx]: console.log?

**Review by steadyGuy — APPROVED** [review]

File: frontend/package.json (hunk 1 above)
Comment (romawht22) [reply frontend/package.json:18]: I think it is, it adds couple unique hooks that will be useful later.

File: frontend/src/components/Map/index.tsx (hunk 2 above)
Comment (Yojuks) [reply frontend/src/components/Map/index.tsx]: you should to delete it

File: frontend/yarn.lock [hunk 3]
```diff
@@ -1592,6 +1592,11 @@
   resolved "https://registry.npmjs.org/@popperjs/core/-/core-2.11.5.tgz"
   integrity sha512-9X2obfABZuDVLCgPK9aX0a/x4jaOEweTTWE2+9sr0Qqqevj2Uv5XorvusThmc9XGYpS9yI+fhh8RTafBtGposw==
 
+"@react-leaflet/core@^1.1.1":
+  version "1.1.1"
+  resolved "https://registry.npmjs.org/@react-leaflet/core/-/core-1.1.1.tgz"
+  integrity sha512-7PGLWa9MZ5x/cWy8EH2VzI4T8q5WpuHbixzCDXqixP/WyqwIrg5NDUPgYuFnB4IEIZF+6nA265mYzswFo/h1Pw==
+
 "@rollup/plugin-babel@^5.2.0":
```
Comment (Yojuks) [inline frontend/yarn.lock:1600]: Do we add this file to gitignore?

File: frontend/src/components/Map/index.tsx [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
-  }
+  // Function that defines coordinates on mouse click
+  // function CoordsFinder() {
+  //   useMapEvents({
+  //     click(e) {
+  //       console.log(e.latlng);
+  //     }
+  //   });
+  //   return null;
+  // }
```
Comment (Yojuks) [inline frontend/src/components/Map/index.tsx:14]: Add TODO coment //TODO - ....

**Review by Yojuks — APPROVED** [review]

File: frontend/src/components/Map/index.tsx [hunk 5]
```diff
@@ -1,8 +1,26 @@
 import React from 'react';
-import { StyledMap } from './styles';
+import { MapContainer, TileLayer } from 'react-leaflet';
 
 function Map() {
```
Comment (dobbi-free) [inline frontend/src/components/Map/index.tsx:4]: I think you should use the arrow function to follow the same code style

**Review by dobbi-free — APPROVED** [review]

**Pull request merged**